### PR TITLE
Deletion nomination

### PR DESCRIPTION
**Pull request by noahwalugembe**
Improved Android notification after deletion nomination unclearly worded, and not localized #2660

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/auth/LoginActivity.java [hunk 1]
```diff
@@ -149,6 +150,10 @@ public void onCreate(Bundle savedInstanceState) {
         } else {
             loginCredentials.setVisibility(View.GONE);
         }
+
+        getWindow().setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_PAN);
+
+
```
Comment (nicolas-raoul) [inline app/src/main/java/fr/free/nrw/commons/auth/LoginActivity.java]: This is unrelated, right?

File: app/src/main/java/fr/free/nrw/commons/delete/DeleteTask.java [hunk 2]
```diff
@@ -161,16 +161,25 @@ protected void onProgressUpdate (Integer... values){
 
     @Override
     protected void onPostExecute(Boolean result) {
+
+        //link to string resources
+        String title_resource = context.getResources().getString(R.string.nominating_for_Deletion);
```
Comment (nicolas-raoul) [inline app/src/main/java/fr/free/nrw/commons/delete/DeleteTask.java]: Why uppercase? Please look at the other string and use the same typographical conventions, thanks :-)

File: app/src/main/java/fr/free/nrw/commons/delete/DeleteTask.java [hunk 3]
```diff
@@ -161,16 +161,25 @@ protected void onProgressUpdate (Integer... values){
...
-            title += ": Success";
-            message = "Successfully nominated " + media.getDisplayTitle() + " deletion.";
+            title += ":"+success_resource;
+            message = successfully_nominated_resource + media.getDisplayTitle() + for_deletion_resource+".";
         }
         else {
-            title += ": Failed";
-            message = "Could not request deletion.";
+            title += ":"+failed_resource;
+            message = could_not_request_deletion_resource+".";
```
Comment (nicolas-raoul) [inline app/src/main/java/fr/free/nrw/commons/delete/DeleteTask.java]: In Japanese the point is a different character, so it should be localized too :-)

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -470,4 +470,12 @@ Upload your first media by tapping on the add button.</string>
   <string name="image_chooser_title">Choose Images to upload</string>
 
   <string name="please_wait">Please wait…</string>
+
+  <string name="nominating_for_Deletion">Nominating for Deletion</string>
+  <string name="success">Success</string>
+  <string name="successfully_nominated">Successfully nominated</string>
+  <string name="for_deletion">for deletion</string>
```
Comment (nicolas-raoul) [inline app/src/main/res/values/strings.xml]: Please read https://github.com/commons-app/apps-android-commons/issues/2498 and fix accordingly.

**Comment by nicolas-raoul** [comment]
Good start, but there are several things to improve :-)

File: app/src/main/java/fr/free/nrw/commons/auth/LoginActivity.java (hunk 1 above)
Comment (noahwalugembe) [reply app/src/main/java/fr/free/nrw/commons/auth/LoginActivity.java]: Hi @nicolas-raoul I branched from my old master branch which has an merged code  is it okay if you can merge those changes so that i can make a new fork

**Comment by noahwalugembe** [comment]
I need some one to help me with this  current conflict in strings.xml

**Comment by madhurgupta10** [comment]
@noahwalugembe I can help you with it

**Comment by noahwalugembe** [comment]
Okay dear please go on

**Comment by madhurgupta10** [comment]
@noahwalugembe I encourage you to do a side by side comparison of [your version of strings.xml](https://github.com/commons-app/apps-android-commons/blob/13fdc48a6d2e5a189ace96da078c33834a10d75c/app/src/main/res/values/strings.xml) with [the current version of strings.xml](https://github.com/commons-app/apps-android-commons/blob/master/app/src/main/res/values/strings.xml) on https://www.diffchecker.com/diff

Then you can resolve the conflicts as explained here in[ this tutorial ](https://www.youtube.com/watch?v=JtIX3HJKwfo)

**Comment by noahwalugembe** [comment]
Thanks am going to work on it

**Comment by noahwalugembe** [comment]
> @noahwalugembe I can help you with it

Thanks dear its okay now

**Comment by madhurgupta10** [comment]
@noahwalugembe Hey, it seems like you are trying to solve two separate issues together in one PR, here how you can solve it, create a new branch by using the command
`git pull origin master`
`git checkout -b <new-branch-name> master`
then delete the unnecessary code and commit again to your new branch
`git commit -m <message>`
`git push origin <new-branch-name>`
and then you can open new PR from the new branch
you can switch branches using the command
`git checkout <branch-name>`
and to know which is the current branch use `git branch` where * represents the current branch.

**Comment by noahwalugembe** [comment]
> @noahwalugembe Hey, it seems like you are trying to solve two separate issues together in one PR, here how you can solve it, create a new branch by using the command
> `git pull origin master`
> `git checkout -b <new-branch-name> master`
> then delete the unnecessary code and commit again to your new branch
> `git commit -m <message>`
> `git push origin <new-branch-name>`
> and then you can open new PR from the new branch
> you can switch branches using the command
> `git checkout <branch-name>`
> and to know which is the current branch use `git branch` where * represents the current branch.

Is it okay if you can merge for me pull request #2618 so that i can delete and then make a new fresh fork
